simple/Makefile

```diff
@@ -19,4 +19,35 @@ endef
 
 $(eval $(call template,alu,gates luts))
 
+# DFF with constant drivers
+$(eval $(call template,dff, dff))
+$(eval $(call template,dffr, dff))
+$(eval $(call template,dffc, dff))
+$(eval $(call template,dffcp, dff))
+$(eval $(call template,dffsr, dff))
```

Comment:
The `dff` doesn't really test a DFF. It only tests the special case of a DFF that has it's D input hard-wired to zero, and so forth. Maybe a more expressive name would be better. Maybe for example `dff_d0`.